MainApp.MVC/Areas/IntranetPortal/Controllers/SpecialActionsController.cs

```diff
@@ -35,6 +41,72 @@ public async Task<IActionResult> Index()
             //    }
             //}
             return View();
-        }       
+        }
+        [HttpPost]
+        public async Task<IActionResult> ResetTranslationCache()
+        {
+            var translationClearCacheIntranetPortalUrl = await _appSettingsAccessor.GetApplicationSettingValueByKey<string>("TranslationClearCacheIntranetPortal", "TranslationClearCacheIntranetPortal");
+            var translationClearCachePublicPortalUrl = await _appSettingsAccessor.GetApplicationSettingValueByKey<string>("TranslationClearCachePublicPortal", "TranslationClearCachePublicPortal");
+
+            StringBuilder stringBuilderSuccess = new StringBuilder();
+            StringBuilder stringBuilderError = new StringBuilder();
+            try
+            {
+                HttpClient client = new HttpClient();
```

Comment:
Netrogatelno, poshto e tuka.
Ali treba da smenime nachin na koj koristime HttpClient.
Vo ovaa situacija, za da ne go komplicirame sega, treba da se izvadi kreiranjeto na HttpClient pred try catch-ot i u finally da se naprai Dispose().
Osobeno koga kje go koristime API-to.
Tamu treba da ideme so HttpClientFactory i od tamu da zemame HttpClient.
Pred toa treba da creirame Named Clients.